app/views/layouts/application.html.erb

```diff
@@ -54,7 +61,7 @@
     <% end %>
     <%= yield %>
     <footer>
-      <img src = "https://ucf03e640a97681ba9054865a0d8.previews.dropboxusercontent.com/p/thumb/AA2J4xfIdpoiKzpEg9lZYaTXlTX3RafcBvvgt2wx8EjZOQplOqrNFjPseeQB7Bl7A7-dH3miZUO0KxtPrMS4ZcqqoiF8C5GvctkYsASv7l0S2RpngrmRH0CPcHB9A3Im9IrGcr8RAjIUNZbkMjm6RKzmjj9EOt2obhvhh5zHCxBXTa4TKQ5Zxy7OYj6purZ8x4pT79plGz5XX76mXGGCcLyT_EDZQf-FaGbtaLefkBpvKl17FMfbujQqMGvPVPNL4m9nU98u6QkwmAIYfmx2xCYbDbYVwYcaEjnS99ENPi17S6vZqavb0KT_CeIoVpDyqOAf4xu7xsF386JhOtgqTqkL6M1s6O1693Wl4rZQIbQAZ_JzOJwzGMDyJeGmv8GEbOcQDUWpfLVWznuDbyuAGYbb/p.png?fv_content=true&size_mode=5" height="30">
+      <img src="api-transparent background for non-white background.png" alt="powered by listen notes", height = "20">
```

Comment:
Updated to image in our repo in place of link
Caution: this still have inline styling, so if anyone ever wants to manipulate it please see the height tag at the end of the link.